src/test/suite/explorer/helpExplorer.test.ts

```diff
@@ -44,11 +44,12 @@ suite('Help Explorer Test Suite', function () {
     const atlasHelpItem = helpTreeItems[5];
 
     assert.strictEqual(atlasHelpItem.label, 'Create Free Atlas Cluster');
-    const telemetryUserIdentity =
+    assert.strictEqual(atlasHelpItem.url.includes('mongodb.com'), true);
+    const { userId, anonymousId } =
       mdbTestExtension.testExtensionController._telemetryService.getTelemetryUserIdentity();
     assert.strictEqual(
-      atlasHelpItem.url,
-      `https://mongodb.com/products/vs-code/vs-code-atlas-signup?utm_campaign=vs-code-extension&utm_source=visual-studio&utm_medium=product&ajs_aid=${telemetryUserIdentity[0]}`
+      new URL(atlasHelpItem.url).searchParams.get('ajs_aid'),
+      userId ?? anonymousId
```

Comment:
It doesn't look like we test that the url has the `utm_source=visual-studio&utm_medium=product&ajs_aid` parameters anymore. Would it be helpful to we add that either to this test or add a unit test for `src/utils/links.ts`?